src/utils/post_utils.js

```diff
@@ -192,24 +202,35 @@ export const postTypePriority = {
     [Posts.POST_TYPES.ADD_TO_CHANNEL]: 5,
     [Posts.POST_TYPES.LEAVE_CHANNEL]: 6,
     [Posts.POST_TYPES.REMOVE_FROM_CHANNEL]: 7,
+    [Posts.POST_TYPES.PURPOSE_CHANGE]: 8,
```

Comment:
Why add these values? This array is only used for sorting posts as part of combined system messages, and the types you've added are not combined into those

Reply:
Is not needed as part of the "Code" itself, but is needed to be consistent and complete with the available post types. Basically is for make flow pass, other way is to define an specific type for "combinable post types", or something like that.

Reply:
Alright, I guess this is fine. I'll likely take another pass through this anyway since I'll be rewriting part of that anyway